base/applications/rapps/available.cpp

```diff
@@ -54,32 +54,32 @@ VOID CAvailableApplicationInfo::RetrieveGeneralInfo(AvailableStrings& AvlbString
     GetString(L"URLSite", m_szUrlSite);
     GetString(L"SHA1", m_szSHA1);
 
-    static_assert(MAX_SNAPSHOT_NUM < 10000, "MAX_SNAPSHOT_NUM is too big");
-    for (int i = 0; i < MAX_SNAPSHOT_NUM; i++)
+    static_assert(MAX_SCRNSHOT_NUM < 10000, "MAX_SCRNSHOT_NUM is too big");
+    for (int i = 0; i < MAX_SCRNSHOT_NUM; i++)
     {
-        WCHAR SnapshotField[sizeof("Snapshot") + 4];
-        wsprintfW(SnapshotField, L"Snapshot%d", i + 1);
-        ATL::CStringW SnapshotLocation;
-        if (!GetString(SnapshotField, SnapshotLocation))
+        WCHAR ScrnshotField[sizeof("Scrnshot") + 4];
+        wsprintfW(ScrnshotField, L"Scrnshot%d", i + 1);
+        ATL::CStringW ScrnshotLocation;
+        if (!GetString(ScrnshotField, ScrnshotLocation))
```

Comment:
```suggestion
        CStringW ScrnshotField;
        ScrnshotField.Format(L"Screenshot%d", i + 1);
        CStringW ScrnshotLocation;
        if (!GetString(ScrnshotField, ScrnshotLocation))
```
There are multiple changes worth noting:
* We can use the CString's capability of formatting text
* The ATL:: prefix is not needed in this case
* I think it is better to use the non-abbreviated version 'screenshot' because this is something that is going to be typed out by people.

Reply:
yes, you are right🤣